docs/components/contexts/DocsExperimentProvider.js

```diff
@@ -0,0 +1,39 @@
+// @flow strict
+import { type Node } from 'react';
+import { ExperimentProvider } from 'gestalt';
+
+/**
+ * To implement experimental behavior in the docs:
+ * - Add your experiment name here
+ * - Unless you want the experimental behavior live on the docs for everyone, REMOVE YOUR EXPERIMENT HERE before merging your PR!
+ * */
+
+const enabledExperiments = [];
+
+function buildExperimentsObj(experiments: $ReadOnlyArray<string>) {
+  return experiments.reduce(
+    (
+      acc: {|
+        [string]: {|
+          anyEnabled: boolean,
+          group: string,
+        |},
+      |},
+      cur: string,
+    ) => ({
+      ...acc,
+      [cur]: { anyEnabled: true, group: 'enabled' },
+    }),
+    {},
+  );
+}
+
+type Props = {| children: Node |};
+
+export default function DocsExperimentProvider({ children }: Props): Node {
+  return (
+    <ExperimentProvider value={buildExperimentsObj(enabledExperiments)}>
```

Comment:
If more specific groups are needed, we can spread those into the object returned from the helper function, or possibly modify the function. Going with the simple solution for now for YAGNI reasons.